Assets/Scripts/Player/PlayerShoot.cs

```diff
@@ -8,15 +8,23 @@ public class PlayerShoot : MonoBehaviour
     {
         public GameObject bulletPrefab;
         public Transform gunEnd;
-        public PlayerAttributesManager attributesManager;
+        private PlayerAttributesManager _attributesManager;
 
         private double _fireTimer;
```

Comment:
change name from _fireTimer to  _shootTimer

Reply:
Done